lib/beautifier.js

```diff
@@ -235,7 +235,12 @@ class Beautifier {
             } else if (_.isObject(value)) {
                 newItem[newKey] = this.beautify(value, beautifications[key]);
             } else {
-                newItem[newKey] = value;
+                let floatVal = parseFloat(value);
+                if (!_.isNaN(floatVal)) {
+                  newItem[newKey] = floatVal;
+                } else {
+                  newItem[newKey] = value;
+                }
```

Comment:
Can you verify that this doesn't unintentionally parse anything that shouldn't be a float? I would prefer if there was a safer way to make sure only the desired properties are processed as a float. Technically something as silly as this would be parsed incorrectly as a float by this logic - so I'm concerned there may be unintended side-effects in this change:

```
parseFloat("0a40a19ce6ddfa4bd0d98284a98794e0ff3b0d04bd265d4fe5272e341cdb7a47R238");
```

Reply:
I have been using this outside the main library without any adverse side effects -a this._floatKeys [] could be introduced if needed.

Reply:
> this._floatKeys [] could be introduced if needed to check the being parsed.

I would prefer that to be honest. It would reduce the risk of unintended side effects now as well as future proofing in expansions of the library. Thanks!